src/lib/__tests__/AnnotationControls-test.js

```diff
@@ -38,222 +32,220 @@ describe('lib/AnnotationControls', () => {
 
     afterEach(() => {
         fixture.cleanup();
-        sandbox.verifyAndRestore();
 
         annotationControls = null;
         stubs = {};
     });
 
     describe('constructor()', () => {
-        it('should create the correct DOM structure', () => {
-            expect(annotationControls.controls).not.to.be.undefined;
-            expect(annotationControls.controlsElement).not.to.be.undefined;
-            expect(annotationControls.currentMode).to.equal(AnnotationMode.NONE);
+        test('should create the correct DOM structure', () => {
+            expect(annotationControls.controls).toBeDefined();
+            expect(annotationControls.controlsElement).toBeDefined();
+            expect(annotationControls.currentMode).toBe(AnnotationMode.NONE);
         });
 
-        it('should throw an exception if controls is not provided', () => {
-            expect(() => new AnnotationControls()).to.throw(Error, 'controls must be an instance of Controls');
+        test('should throw an exception if controls is not provided', () => {
+            expect(() => new AnnotationControls()).toThrowError(Error);
```

Comment:
Does the error message need to be validated?

Reply:
Probably not? 😄